packages/modular-scripts/package.json

```diff
@@ -152,8 +159,8 @@
   "devDependencies": {
     "@modular-scripts/modular-types": "1.2.0",
     "@schemastore/package": "0.0.6",
-    "@schemastore/tsconfig": "0.0.9",
```

Comment:
Why is this removed?

Reply:
Not sure, must have been unintentional - any reason we have duplicate dependencies in devDependencies and normal dependencies?